src/include/meta.h

```diff
@@ -98,93 +97,3 @@ struct __wt_blkincr {
                               /* AUTOMATIC FLAG VALUE GENERATION STOP 8 */
     uint8_t flags;
 };
-
-/*
- * Block modifications from an incremental identifier going forward.
- */
-/*
- * At the default granularity, this is enough for blocks in a 2G file.
- */
-#define WT_BLOCK_MODS_LIST_MIN 128 /* Initial bits for bitmap. */
-struct __wt_block_mods {
```

Comment:
`WT_BLOCK_MODS` seems to be used in src/block as well, it should not be owned by the checkpoint but the block module instead?

Reply:
@sueloverso can you please comment?

Reply:
Commented above - that is a sub-structure of the WT_CKPT.